topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.md

```diff
@@ -637,26 +686,23 @@ Let's go ahead and classify those sequences using the Bayesian classifier with t
 > {: .question}
 {: .hands_on}
 
-Also of note is that *unknown* only pops up as a classification if the classifier cannot classify your
-sequence to one of the domains.
-
-At this point we have curated our data as far as possible and we're ready to see what our error rate is.
+The data is now as clean as we can get it.  Despite that, our remaining sequences will still have errors in them.
 
-
-## Optional: Assessing error rates based on our mock community
+## Optional: Calculate error rates based on our mock community
 
 
 > ### {% icon tip %} Tip: Skipping the mock community analysis
 >
-> The mock community analysis is optional. If you are low on time or want to skip ahead, you can jump straight to [the next section](#preparing-for-analysis)
+> The mock community analysis is optional. If you are low on time or want to skip ahead, you can jump straight to [the next section](#removing-mock-sample)
 > where we will cluster our sequences into OTUs, classify them and perform some visualisations.
 >
 {: .tip}
 
-Measuring the error rate of your sequences is something you can only do if you have co-sequenced a mock
-community, that is, a sample of which you know the exact composition. This is something we include for
-every 95 samples we sequence. You should too because it will help you gauge your error rates and allow
-you to see how well your curation is going, and whether something is wrong with your sequencing setup.
+
+The following step is only possible if you have co-sequenced a mock community with your samples. A mock community is a sample
+of which you know the exact composition. This is something we recommend you to do, because it will give you an idea of how
```

Comment:
A mock community is a sample of which you know the exact composition and is recommended to do, because it will ...